test/libponyc/badpony.cc

```diff
@@ -626,8 +626,35 @@ TEST_F(BadPonyTest, DefaultArgScope)
 TEST_F(BadPonyTest, GenericMain)
 {
   const char* src =
-    "actor Main[X]"
+    "actor Main[X]\n"
     "  new create(env: Env) => None";
 
   TEST_ERRORS_1(src, "the Main actor cannot have type parameters");
 }
+
+TEST_F(BadPonyTest, AsBadCap)
+{
+  const char* src =
+    "actor Main\n"
+    "  new create(env: Env) =>\n"
+    "    let v: I64 = 3\n"
+    "    try (v as I64 iso) end";
+
+  TEST_ERRORS_1(src, "this capture violates capabilities");
+}
+
+TEST_F(BadPonyTest, AsUnaliased)
```

Comment:
The intent of this test is unclear to me. This code compiles fine under the current version of the compiler.